apps/condo/domains/user/integration/UserExternalIdentityMiddleware.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express')
+
+const { expressErrorHandler } = require('@condo/domains/common/utils/expressErrorHandler')
+const { SbbolRoutes } = require('@condo/domains/organization/integrations/sbbol/routes')
+const { SberIdRoutes } = require('@condo/domains/user/integration/sberid/routes')
+
+class UserExternalIdentityMiddleware {
+    async prepareMiddleware ({ keystone }) {
+        const app = express()
+
+        // sbbol route
+        const sbbolRoutes = new SbbolRoutes()
+        app.get('/api/sbbol/auth', sbbolRoutes.startAuth.bind(sbbolRoutes))
+        app.get('/api/sbbol/auth/callback', sbbolRoutes.completeAuth.bind(sbbolRoutes))
+
+        // sber_id route
+        const sberIdRoutes = new SberIdRoutes()
+        app.get('/api/sber_id/auth', sberIdRoutes.startAuth.bind(sberIdRoutes))
```

Comment:
It’s better not to use underscor inside url.

Reply:
/api/sber_id/callback already configured on sber id side